cpp-linter/src/clang_tools/clang_tidy.rs

```diff
@@ -178,14 +179,15 @@
                 // likely not a member of the project's sources (ie /usr/include/stdio.h)
                 filename = filename
                     .strip_prefix(&cur_dir)
-                    .expect("cannot determine filename by relative path.")
+                    // we already checked above that filename.starts_with(current_directory)
+                    .unwrap()
```

Comment:
Again, read the comment:

> ```rs
>              // we already checked above that filename.starts_with(current_directory)
> ```

`unwrap()` here is fine because the block is not entered if it can fail:
> ```rs
> if filename.is_absolute() && filename.starts_with(&cur_dir) {
> ```